package.json

```diff
@@ -50,7 +50,7 @@
     "vitest": "0.32.2"
   },
   "dependencies": {
-    "@xhmikosr/downloader": "^12.0.0",
+    "@xhmikosr/downloader": "^13.0.0",
     "node-fetch": "^3.3.1",
     "semver": "^7.5.1"
```

Comment:
Not related to this PR but might be worth bumping this to 7.5.3 since the newer versions fix security vulnerabilities.

Reply:
thanks